src/radical/pilot/agent/rm/base.py

```diff
@@ -197,6 +197,7 @@ def __init__(self, cfg, session):
     @classmethod
     def create(cls, name, cfg, session):
 
+        # pylint: disable=protected-access
```

Comment:
What warning is addressed with this actually?  Just `session._log`?  Then this should be fixed by adding a log getter to the session I guess.

Reply:
The same as above. Let's use linters to learn better coding practices. I think respecting private variables is good practice. Would you all agree?